homeassistant/components/zwave.py

```diff
@@ -371,10 +374,22 @@ def scene_activated(node, scene_id):
             ATTR_SCENE_ID: scene_id
         })
 
+    def node_event_activated(node, value):
+        """Called when a nodeevent is activated on any node in the network."""
+        name = _node_name(node)
+        object_id = "{}_{}".format(slugify(name), node.node_id)
+
+        hass.bus.fire(EVENT_NODE_EVENT, {
+            ATTR_ENTITY_ID: object_id,
+            ATTR_VALUE_ID: value
```

Comment:
Please make a new attribute. Preferably ATTR_BASIC_LEVEL =  "basic_level" which is what it represent.